external/rootpy

```diff
@@ -1 +1 @@
-Subproject commit f0f123d5ebce3983ef3bd3d5d8c962787ca499b4
+Subproject commit 295bfb2624a3bf286348f61de009d0c0f3dfb323
```

Comment:
@kreczko I can't remember if this should ever be changed?  Or maybe I updated it, and this is undoing it?